h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -1204,18 +1205,19 @@ private Frame encodeFrameCategoricals(Frame fr, boolean scopeTrack) {
   /**
    * Rebalance a frame for load balancing
    * @param original_fr Input frame
-   * @param local Whether to only create enough chunks to max out all cores on one node only
+   * @param local Whether to only create enough chunks to max out all cores on one node only; Warning: IGNORED in the code!
```

Comment:
Technically not true, it is pass to desiredChunks function.

Reply:
Good point, function "desiredChunks" can be overridden by 3rd party algos and used somehow. rebalance can also be overridden.

I will change it, thanks